packages/react-workspace/agents.md

```diff
@@ -0,0 +1,19 @@
+# Nimble React – AI Instructions
+
+## Overview
+React wrappers for Nimble Web Components.
+- **Status**: Alpha / Experimental
+- **Pattern**: `useLayoutEffect` for events, `forwardRef` for DOM access.
+
+## Key References
+- `README.md` – Current status.
+
+## Core Patterns
+- Ensure custom elements are registered before use.
+- Use `useLayoutEffect` to attach event listeners to custom events.
+- Forward refs to the underlying DOM element.
```

Comment:
first two are incorrect, those are not how wrappers are implemented. Should just point to like banner wrapper as an example

last one really only applies to the example app, and is just a statement of React's features. Don't see why we call out that specific react feature, seems unnecessary